specs/UserActivityHistory/UserActivityHistoryAPI.md

```diff
@@ -3,21 +3,43 @@ UserActivityHistory
-interacting with.
+This allows Windows to have insight into the application state, enabling smart experiences that are
+built around the semantics of the app. For example, a document editor can give Windows information
+about the document that the user is editing, so that Recall can later take the user to the document
+at the same location.
+
+To record user activity, an app uses [UserActivityChannel](https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.useractivities.useractivitychannel?view=winrt-26100)
+to retrieve a UserActivity object via the API [GetOrCreateUserActivityAsync](https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.useractivities.useractivitychannel.getorcreateuseractivityasync?view=winrt-26100#windows-applicationmodel-useractivities-useractivitychannel-getorcreateuseractivityasync(system-string)).
+If a UserActivity with the given ID already exists, it will be returned; otherwise, a new UserActivity
+object will be created and returned. You can then call the API [GetSession](https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.useractivities.useractivity.createsession?view=winrt-26100#windows-applicationmodel-useractivities-useractivity-createsession)
+to return a [UserActivitySession](https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.useractivities.useractivitysession?view=winrt-26100)
+object that tracks how long the user is engaged in that activity. This structure allows multiple
+sessions to be associated with the same activity, representing the case where the user completes
+that activity a bit at a time - e.g., beginning to watch a movie, then pausing, then watching more later.
+These will be treated as the same singular user activity that spans multiple sessions.
+
+UserActivityHistory is a new set of APIs that allow you to query up to the past 28 days of the
+user's activity history, which will enable you to bring back content that the user has previously
+been interacting with.
+
+# Conceptual pages (How To)
+
+The intended use case of this API is to allow you to make search queries against the user's activity history
+on their local computer.  The string matching in the query parameters in this API is lexical in nature,
+meaning that it is expected that any natural language semantic parsing of the user's input will be done by
+your app prior to calling this API.
+
+For example, if a user types in something along the lines of, "Please find the Korean recipe I was looking at
+earlier today", your app might have an agentic AI parse that input and determine that the user is looking
+for a webpage that contains the keywords "Korean" and "recipe", and construct a query with those keywords,
+a content type of "text/html", and an access time within the last 24 hours.
+
+In order for an app to make use of this API, it must be Windows logo certified, and the user must provide
```

Comment:
link to "how do I get the Windows logo certification" page?

Reply:
Also, need to take into account whatever consent model we end up with - one time, every time, does it include context, etc?